src/oic/oic/message.py

```diff
@@ -914,6 +967,119 @@ class ResourceRequest(Message):
     "offline_access": []
 }
 
+# LOGOUT related messages
+
+SINGLE_OPTIONAL_JSON = ParamDefinition(dict, False, json_ser, json_deser, False)
+SINGLE_REQUIRED_JSON = ParamDefinition(dict, True, json_ser, json_deser, False)
+
+BACK_CHANNEL_LOGOUT_EVENT = "http://schemas.openid.net/event/backchannel-logout"
+
+
+class LogoutToken(Message):
+    """
+    Defined in
+    https://openid.net/specs/openid-connect-backchannel-1_0.html#LogoutToken
+    """
+    c_param = {
+        "iss": SINGLE_REQUIRED_STRING,
+        "sub": SINGLE_OPTIONAL_STRING,
+        "aud": REQUIRED_LIST_OF_STRINGS,  # Array of strings or string
+        "iat": SINGLE_REQUIRED_INT,
+        "jti": SINGLE_REQUIRED_STRING,
+        'events': SINGLE_REQUIRED_JSON,
+        'sid': SINGLE_OPTIONAL_STRING
+    }
+
+    def verify(self, **kwargs):
+        super(LogoutToken, self).verify(**kwargs)
+
+        if 'nonce' in self:
+            raise MessageException('"nonce" is prohibited from appearing in '
+                                   'a LogoutToken.')
+
+        # Check the 'events' JSON
+        _keys = list(self['events'].keys())
+        if len(_keys) != 1:
+            raise ValueError('Must only be one member in "events"')
+        if _keys[0] != BACK_CHANNEL_LOGOUT_EVENT:
+            raise ValueError('Wrong member in "events"')
+        if self['events'][_keys[0]] != {}:
+            raise ValueError('Wrong member value in "events"')
+
+        # There must be either a 'sub' or a 'sid', and may contain both
+        if not ('sub' in self or 'sid' in self):
+            raise ValueError('There MUST be either a "sub" or a "sid"')
+
+        try:
+            if kwargs['aud'] not in self['aud']:
+                raise NotForMe('Not among intended audience')
+        except KeyError:
+            pass
+
+        try:
+            if kwargs['iss'] != self['iss']:
+                raise NotForMe('Wrong issuer')
+        except KeyError:
+            pass
+
+        _now = utc_time_sans_frac()
+
+        try:
```

Comment:
Can be replaced by a single line `_skew = kwargs.get("skew", 0)`, same for `iat`.